cardpresent/src/main/java/com/fattmerchant/omni/UserNotificationListener.kt

```diff
@@ -0,0 +1,43 @@
+package com.fattmerchant.omni
+
+import com.fattmerchant.omni.data.UserNotification
+import com.creditcall.chipdnamobile.UserNotification as ChipDnaUserNotification
```

Comment:
What do you think about avoiding the ChipDna dependency here? I believe the rest of com.fattmerchant.omni is purely omni business logic removed from ChipDna and AWC which are within the com.fattmerchant.android package. I feel like this is a pretty big refactor but I also feel it's worth the trouble because keeping chipdna out of scope here would allow us to test the omni logic without having to test chipdna. It would also provide a generic api for the consuming app

Reply:
Actually the refactor doesn't seem big. The change i'm proposing is to have onUserNotification return the com.fattmerchant.omni.data.UserNotification instead of the ChipDNA one. Let's talk about it and see how we move forward so I can do the same thing on iOS

Reply:
The only reason that chip dna is imported is for documentation purposes. If that isn't necessary I can remove it.